doc/man7/provider-signature.pod

```diff
@@ -194,15 +194,17 @@ algorithms:
 
 =over 4
 
-=item "digest" (B<OSSL_SIGNATURE_PARAM_DIGEST>) <utf8 string>
+=item "digest" (B<OSSL_SIGNATURE_PARAM_DIGEST>) <UTF8 string>
```

Comment:
We do seem to have 3 notations used in different files currently..
 =item "digest" (B<OSSL_SIGNATURE_PARAM_DIGEST>)
=item B<OSSL_MAC_PARAM_SIZE>
=item B<OSSL_MAC_PARAM_SIZE> ("size")

I agree that it would be good to be consistent..

Reply:
The third is the most common.